src/pincode/authentication.test.ts

```diff
@@ -51,6 +52,16 @@ jest.mock('src/utils/sleep', () => ({
   sleep: jest.fn().mockResolvedValue(true),
 }))
 
+jest.mock('src/utils/aes', () => {
+  const originalAes = jest.requireActual('src/utils/aes')
+
+  return {
+    ...originalAes,
```

Comment:
Why do we need to return the original import here? It seems like the only two functions in that file are `aesEncrypt` and `aesDecrypt`, which we're mocking anyways.